src/components/Home/Home.js

```diff
@@ -17,49 +20,98 @@ function Home () {
   useEffect(() => {
     async function position () {
       try {
-        await navigator.geolocation.getCurrentPosition(function (position) {
-          setLatitude(position.coords.latitude)
-          setLongitude(position.coords.longitude)
-        })
+        const now = new Date()
+        if (JSON.parse(localStorage.getItem('location') === null) || now.getTime() > JSON.parse(localStorage.getItem('location')).expiryTime) {
```

Comment:
At so many places, we are doing `JSON.parse(localStorage.getItem('location')`. Can we also make a separate module for this? Let's say `utils/storage` in this case